winpr/libwinpr/sspi/NTLM/ntlm.c

```diff
@@ -114,24 +110,19 @@ int ntlm_SetContextServicePrincipalNameA(NTLM_CONTEXT* context, char* ServicePri
 int ntlm_SetContextTargetName(NTLM_CONTEXT* context, char* TargetName)
 {
 	int status;
```

Comment:
@bmiklautz: Not your changes, but this function looks quite broken (name <--> TargetName), etc.